### DISCUSSION TITLE
Difference between offset first and offset all?

### DISCUSSION THREAD

**Discussion by AlmightyWings232** [xgi/houdoku#251] [Q&A] [answered]
Just the title, what's the difference between the 'Offset first page' and 'Offset all' setting in the reader? Coz I don't notice any

Also, the UI has improved a lot since I last used the program. Shoutout to whoever designed it

**Comment by xgi** [comment, marked as answer]
In the double-page view, spread pages (large images that would've been spread across both pages in an actual book) are displayed by themself, which looks more accurate.

With "offset all", pages after a spread are also offset. With "offset first", only the first page is offset, ignoring any spreads. If you don't encounter spread pages then you can use either, but you may have to change this setting depending on how the scanlator organized their pages.

**Reply by AlmightyWings232** [reply]
Aaah, right. That makes sense now (sorry for the hella late reply tho 😅)
